website/content/docs/intro/index.mdx

```diff
@@ -49,71 +41,50 @@ Review the video below to learn more about Consul from HashiCorp's co-founder Ar
-[here](/docs/architecture/anti-entropy#catalog).
-
-Components of your infrastructure that need to discover other services
-or nodes can query any of the Consul servers _or_ any of the Consul agents.
-The agents forward queries to the servers automatically.
-
-Each datacenter runs a cluster of Consul servers. When a cross-datacenter
-service discovery or configuration request is made, the local Consul servers
-forward the request to the remote datacenter and return the result.
-
-## Next Steps
-
-Continue onwards with [HashiCorp Learn](https://learn.hashicorp.com/tutorials/consul/get-started-install)
-to learn more about Consul and how to get Consul up and running.
+### Automate service discovery
+
+Adopting a microservices architecture on cloud infrastructure is a critical step toward delivering value at scale, but knowing where healthy services are running on your networks in real time becomes a challenge. Consul automates service discovery by replacing service connections usually handled with load balancers with an identity-based service catalog. The service catalog is a centralized source of truth that you can query through Consul’s DNS server or API. The catalog always knows which services are available, which have been removed, and which services are healthy.
+
+### Connect services across runtimes and cloud providers
+
+The shift to cloud has evolved. Organizations have adopted a multi-cloud strategy where customer value is delivered on various runtimes and cloud providers, which can lead to inconsistency and technical debt. Consul routes network traffic to any runtime or infrastructure environment your services need to reach using proxies. You can also use Consul API Gateway to route traffic into and out of the network. Consul service mesh provides additional capabilities, such as securing communication between services, traffic management, and observability, with no application code changes.
+
+Consul also has many integrations with Kubernetes that enable you to leverage Consul features in containerized environments. For example, Consul can automatically inject sidecar proxies into Kubernetes Pods and sync Kubernetes Services and non-Kubernetes services into the Consul service registry. 
+
+You can also schedule Consul workloads with [HashiCorp Nomad](https://www.nomadproject.io/) to provide secure service-to-service communication between Nomad jobs and task groups. 
+
+### Enable zero-trust network security
+
+Microservice architectures expose more of a network’s surface area, which demand additional security measures to prevent unwanted access by malicious actors. Consul provides several mechanisms that enhance network security without any changes to your application code, including mutual transport layer security (mTLS) encryption on all traffic between services and  Consul intentions, which are service-to-service permissions that you can manage through the Consul UI, API, and CLI.
```

Comment:
I think it's less Microservice architectures expose more of a network's surface area, and more 1) additional complexity leads to increased difficulty in securing services; and 2) traditionally doing security right within an application is one of the harder things to do.

"More complex and more dynamic architectures lead to an increased difficulty in securing services and protecting against accidental disclosure and malicious actors. And that is on top of doing security right has traditionally been one of the more difficult things for an application to do. The Consul service mesh provides several mechanisms to enhance network security without any changes to your application code by providing a network fabric which handles that for any applications using the mesh. Things like mutual transport layer security (mTLS) encryption of all traffic, provisioning and managing mTLS certificates, and authorizing service-to-service communication through Consul intentions, are handled for applications. This allows your development teams to focus on the core business logic of their service, knowing they'll have a consistent and secure way to communicate handled for them."